src/components/Navigation.js

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
-  return (
-    <div>
-      <div className="App">
-        <h1>React Router Mini</h1>
+    return (
         <div>
-          <a href="">Home</a>
+            <div className="App">
+                <h1>React Router Mini</h1>
+                <div>
+                    <Link to="/">Home</Link>
+                </div>
+                <div>
+                    <Link to="/about">About</Link>
+                </div>
+                <div>
+                    <Link to="/contact">Contact</Link>
+                </div>
```

Comment:
An interesting alternative to repeating yourself with these `Link`s in `div`s would be making a function for these:
```javascript
const makeLink = text => (
  <div>
    <Link to=`/${text}`>
      { text[0].toUpperCase() + text.slice(1) }
    </Link>
  </div>
);
```
and then, for the latter two at least, you could integrate it like this:
```javascript
const Navigation = () => (
  <div>
    <div className="App">
      <div>
        <Link to="/">Home</Link>
      </div>
      { makeLink('about') }
      { makeLink('contact') }
    </div>
  </div>
);
```

Reply:
another option would be having _two_ parameters; something like this:
An interesting alternative to repeating yourself with these `Link`s in `div`s would be making a function for these:
```javascript
const makeLink = (text, to='') => (
  <div>
    <Link to=`/${to}`>
      { text }
    </Link>
  </div>
);
```
and then:
```javascript
const Navigation = () => (
  <div>
    <div className="App">
      { makeLink('Home') }
      { makeLink('About', 'about') }
      { makeLink('Contact', 'contact') }
    </div>
  </div>
);
```